app/src/main/kotlin/cz/covid19cz/erouska/exposurenotifications/ExposureNotificationsRepository.kt

```diff
@@ -259,14 +275,46 @@ class ExposureNotificationsRepository @Inject constructor(
     }
 
     suspend fun checkExposure(context: Context) {
-        val lastExposure = getLastRiskyExposure()?.daysSinceEpoch
-        val lastNotifiedExposure = prefs.getLastNotifiedExposure()
-        if (lastExposure != null && lastExposure != lastNotifiedExposure) {
+        importLegacyExopsures()
+        db.dao().deleteOld()
+        db.dao().insert(getDailySummariesFromApi(true).map {
+            DailySummaryEntity(
+                daysSinceEpoch = it.daysSinceEpoch,
+                maximumScore = it.summaryData.maximumScore,
+                scoreSum = it.summaryData.scoreSum,
+                weightenedDurationSum = it.summaryData.weightedDurationSum,
+                importTimestamp = System.currentTimeMillis(),
```

Comment:
to discuss: keby niekto menil cas na telefone, tak tie timestampy nebudu sediet. prevenciou by bolo synchronizovat si cas https://github.com/wandera/truetime-android

Reply:
to je pravda, mohli bychom přejít na nějaké robustnější řešení, na druhou stranu takových uživatelů jsou imho maximálně jednotky

Reply:
@vandac Smooth :slightly_smiling_face: